docs/docs/07-forms.md

```diff
@@ -35,7 +35,7 @@ Like all DOM events, the `onChange` prop is supported on all native components a
 
 ## Controlled Components
 
-An `<input>` with `value` set is a *controlled* component. In a controlled `<input>`, the value of the rendered element will always reflect the `value` prop. For example:
+An `<input>` with a `value` prop is a *controlled* component:
```

Comment:
I kind of like the older wording a little better for this paragraph.  The new wording almost makes it sound like it's a different kind of component, when in fact, it's still an `input` component, but behaves a tiny bit differently.  I like most of the changes in this diff, but this one isn't clearly better IMO.  I especially like that the old wording gave you a hint as to why you might want such a thing before looking at the code.  Otherwise, you read the code without context and are like "wtf?" and then don't figure it out until the next paragraph and then go back and look at the code again.

If we took the old paragraph, but changed the word "set" with "prop specified", would that clear up most of your dislike for that sentence?